src/PowerShellEditorServices/Language/LanguageService.cs

```diff
@@ -7,13 +7,12 @@
 using System;
 using System.Collections.Generic;
 using System.Linq;
+using System.Management.Automation;
+using System.Threading;
 using System.Threading.Tasks;
 
 namespace Microsoft.PowerShell.EditorServices
 {
-    using System.Management.Automation;
-    using System.Management.Automation.Runspaces;
-    using System.Threading;
     /// <summary>
```

Comment:
Ugh, I hope that wasn't me.  I have to watch were Resharper wants to add using statements.